randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -1104,9 +1104,13 @@ Extra - room_id: [33]
   * Extra - door_idx: (73,)
   > Beside missile tank
       Any of the following:
-          High Jump or Screw Attack or Space Jump or Can Freeze Enemies
+          Screw Attack or Space Jump or Can Freeze Enemies
+          All of the following:
+              # dmg expectations
+              High Jump
+              Combat (Intermediate) or Normal Damage ≥ 34
```

Comment:
If you are accounting for a beginner player not being able to make this jump with HJ this should use a movement trick vs combat